thrift0.13/src/main/java/com/linecorp/armeria/internal/common/thrift/ThriftFunction.java

```diff
@@ -346,14 +347,16 @@ public boolean setException(TBase<?, ?> result, Throwable cause) {
         final String ifaceTypeName = typeName(type, funcClass, methodName, "Iface");
         try {
             final Class<?> ifaceType = Class.forName(ifaceTypeName, false, funcClass.getClassLoader());
+
+            // Check and convert to camel, thrift java compiler only support underscored to camel
+            final String methodNameCamel = methodName.indexOf('_') != -1 ?
```

Comment:
How about using `>= 0` instead of `!= -1`? I somehow feel like it's less confusing.